koku/api/iam/test/tests_customer_view.py

```diff
@@ -141,15 +174,44 @@ def test_delete_customer(self):
         # Sanity check that the customer count was reduced by 1
         self.assertEquals(Customer.objects.all().count(), customer_count - 1)
 
-    def test_delete_customer_with_reg_users(self):
-        """Test deleting customers with regular users."""
+    def test_delete_customer_with_reg_users_providers(self):
+        """Test deleting customers with regular users and providers."""
         customers = self.customers
         self.assertIsNotNone(customers)
 
         self.assertEquals(len(customers), 2)
         customer_1 = customers[0]
         customer_2 = customers[1]
 
+        # Setup Providers
+        iam_arn = 'arn:aws:s3:::my_s3_bucket'
+        bucket_name = 'my_s3_bucket'
+        token = customer_1['owner']['token']
+        response = self._create_provider(bucket_name, iam_arn, token)
+        self.assertEqual(response.status_code, 201)
+
+        bucket2_name = 'my_other_s3_bucket'
+        response = self._create_provider(bucket2_name, iam_arn, token)
+        self.assertEqual(response.status_code, 201)
+
+        # Verify providers are created
+        provider_url = reverse('provider-list')
+        client = APIClient()
+        client.credentials(HTTP_AUTHORIZATION=token)
+        response = client.get(provider_url)
+        self.assertEqual(response.status_code, 200)
+        json_result = response.json()
+        results = json_result.get('results')
+        self.assertIsNotNone(results)
+        self.assertEqual(len(results), 2)
+        provider_uuids = []
+        for provider in results:
+            provider_uuids.append(provider['uuid'])
```

Comment:
I know some people fine them intolerable, but I ❤️ comprehensions. No need to change this, but did you know you can do

```
provider_uuids = [provider['uuid'] for provider in results]
```

It initializes and populates the variable.